.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+
+Please review GitHub's overview article,
+["Tracking Your Work with Issues"][about-issues].
+
+### Pull Requests
+
+See [Pull Requests](https://github.com/microbiomedata/nmdc-schema/pulls/) for all pull requests. Every pull request should be associated with an issue.
+
+Please review GitHub's article, ["About Pull Requests"][about-pulls],
+and make your changes on a [new branch][about-branches].
+
+We recommend also reading [GitHub Pull Requests: 10 Tips to Know](https://blog.mergify.com/github-pull-requests-10-tips-to-know/)
+
+## GitHub Best Practice
+
+- Read ["About Issues"][about-issues] and ["About Pull Requests"][about-pulls]
+- Issues should be focused and actionable
+- Bugs should be reported with a clear description of the problem and steps to reproduce.  If bugs are found within a notebook, please include the link to the notebook in the issue and the specific cell that is causing the issue.
+- Complex issues should be broken down into simpler issues where possible
+- Pull Requests (PRs) should be atomic and aim to close a single issue
+- PRs should reference issues following standard conventions (e.g. “Fixes #123”)
+- Never work on the main branch, always work on an issue/feature branch
+- Core developers can work on branches off origin rather than forks
+- If possible create a draft or work-in-progress PR on a branch to maximize transparency of what you are doing
+- PRs should be reviewed and merged in a timely fashion
+- In the case of git conflicts, the contributor should try and resolve the conflict
+
+
+[about-branches]: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches
+[about-issues]: https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues
+[about-pulls]: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests
+
+## Adding new notebooks
+
+To add a new notebook to this repository:
+
+1.  Create a folder in the base directory 
+    - Name the folder with a short version of the analysis/question that will be explored.
+    - Make name of folder `snake_case`
+2. Create a `README.md` in the folder outlining the analysis or question.
+3. Create a sub-folder for each language that will be demonstrated
+    - e.g. one subfolder named `R` and one subfolder named `python`
+4. Instantiate a Jupyter Notebook for each folder coded in its corresponding language
+_or_
+4. Create a .Rmd and convert it to a Jupyter Notebook. Several methods for this exist and none are perfect, but [this open source method](https://github.com/mkearney/rmd2jupyter) currently works.
+
```

Comment:
Should we add a bit about creating a link to a readable view of the notebook in the Readme file of the notebook? E.g. using nbviewer, etc.?